docs-wiki/README.md

```diff
@@ -0,0 +1,184 @@
+# 🛠️ FinOps toolkit source
+
+**Welcome aboard!** 🎉 If this is your first time to our repo, here are a few tips:
+
+- Every folder has a README that explains its purpose.
+- If you want to know how to deploy a FinOps toolkit solution, start with the [documentation](https://aka.ms/finops/toolkit).
+- If you want to know how you can contribute, check out the [contribution guide](https://github.com/microsoft/finops-toolkit/tree/dev/CONTRIBUTING.md).
+- If you want to get started with the code, start in the [wiki](https://github.com/microsoft/finops-toolkit/wiki). &nbsp; **← YOU ARE HERE**
+- Read about our [[Architecture]] for context on technologies and structure.
+- Review our [[Coding guidelines]] before you write/review code.
+- Review the guidance below for how to contribute code.
+- When you're ready to dig into code, you'll check the [src](https://github.com/microsoft/finops-toolkit/tree/dev/src) folder.
+
+<br>
+
+On this page:
+
+- [🛫 Get started](#-get-started)
+- [📋 Prerequisites](#-prerequisites)
+- [🗃️ Fork and clone](#️-fork-and-clone)
+- [🌿 Select a branch](#-select-a-branch)
+- [🚀 Deploy](#-deploy)
+- [🔬 Test and verify](#-test-and-verify)
+- [👀 Pull requests](#-pull-requests)
+
+---
+
+## 🛫 Get started
+
+There are many ways to contribute to the FinOps toolkit project, like reporting issues, suggesting features, and submitting or reviewing pull requests. For an overview, refer to the [contribution guide](../CONTRIBUTING.md). This page covers how to contribute to the code.
+
+After cloning and building the repo, check out the [issues list](../issues):
+
+- [`Help wanted ✨`](../issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted+✨%22+sort%3Areactions-%2B1-desc) are areas we'd like explicit help on.
```

Comment:
Might need to adjust the target URI, relative URIs don't work well in Wiki.

Reply:
This one should work since wiki pages are at "finops-toolkit/wiki/Page". In that case "../issues" would open "finops-toolkit/issues", which is correct.